pennylane_lightning/_version.py

```diff
@@ -17,4 +17,4 @@
 """
 
 
-__version__ = "0.30.0-dev7"
+__version__ = "0.30.0.rc0"
```

Comment:
Is there any special reason to use a `.` here instead of `-`, like:
```suggestion
__version__ = "0.30.0-rc0"
```

Reply:
Same comment.

Reply:
Will opt for "-" throughout.

Reply:
Thank you.